src/allmydata/scripts/startstop_node.py

```diff
@@ -6,27 +6,31 @@
 from allmydata.util.encodingutil import listdir_unicode, quote_output
 
 
-class StartOptions(BasedirOptions):
+class StartStopCommonOptions(BasedirOptions):
+    optParameters = [
+        ["pidfile", None, None, "Tell the node which pid file should use."],
```

Comment:
Please add a comment on whether this can be relative (and to what) or absolute. I have some more comments related to that below, and I'm not sure if they are valid.